internal/pkg/kernel/kernel.go

```diff
@@ -111,7 +111,7 @@ func ListKernels() ([]string, error) {
 
 func Build(kernelVersion string, kernelName string, root string) (string, error) {
 	kernelDrivers := path.Join(root, "/lib/modules/"+kernelVersion)
-  kernelDriversRelative := path.Join("/lib/modules/"+kernelVersion)
+	kernelDriversRelative := path.Join("/lib/modules/" + kernelVersion)
```

Comment:
This should be path.Join("/lib/modules", kernelVersion). The use of Join circumvents the need to cat the filename to the main path.

Reply:
I will fix this in another PR as I think it was my error to begin with. Great catch!

Reply:
addressed in #243.